tests/unit/lib/hotkeyManager.spec.ts

```diff
@@ -74,15 +65,17 @@ it("unregisterできる", () => {
     },
   ]);
   hotkeyManager.register(action);
-  expect(dummyHotkeysJs.registeredHotkeys).toEqual([
-    { key: "1", scope: "talk" },
-  ]);
+  hotkeyManager.keyInput(createDummyInput("1", "Digit1") as KeyboardEvent);
+  expect(testCount).toBe(1);
+
   hotkeyManager.unregister(action);
-  expect(dummyHotkeysJs.registeredHotkeys).toEqual([]);
+  hotkeyManager.keyInput(createDummyInput("1", "Digit1") as KeyboardEvent);
+  expect(testCount).toBe(1);
```

Comment:
もしかしたら意図がわかりづらいかも。
なにかメモを残すとかどうでしょう。
```suggestion
  expect(testCount).toBe(1); // 呼び出し回数が増えない
```